src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedIntoPrisonEventService.kt

```diff
@@ -41,20 +42,23 @@ class PrisonerReceivedIntoPrisonEventService(
 
   private fun PrisonerReceivedAdditionalInformation.processPrisonerAdmissionEvent(eventOccurredAt: Instant) {
     log.info { "Processing Prisoner Admission Event for prisoner [$nomsNumber]" }
-
+    val prisoner = prisonerSearchApiService.getPrisoner(nomsNumber)
+    val prisonId = prisoner.prisonId ?: "NA"
```

Comment:
This is the only controversial thing in this PR... prisonId is optional so in theory could be missing so rather than failing here pop in NA?

Reply:
Yes, agreed, though elsewhere we've used `N/A` (see `CreateInitialReviewScheduleMapper` and `TimelineEventFactory`)
I think for consistency we should use `N/A` here as well?

Reply:
ok done